tests/test_rootfs.py

```diff
@@ -166,8 +166,12 @@ def test_expected_files_ext234_mender_configure(
     ):
         """Test mender-configure expected files"""
 
-        # Expect to be installed for Yocto and not for mender-convert
-        expect_installed = not conversion
+        # Expect files to always be installed with our Yocto test images, and on
+        # mender-convert if enabled.
```

Comment:
While this is correct, I wonder 1) why are we introducing it (in CI we are still not installing the addons on any image that we test) and 2) why don't we add the same logic to `test_expected_files_ext234_mender_connect` above

Reply:
You are correct. Up to the very final moments of debugging and fixing this, I thought that the installation of the components is what triggered this. But it's actually only the link creation, so this can in fact be closed. I didn't think about when I put my fixes together in the rebase. Thanks for an observant eye!